### PR TITLE
chore: update all deps

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

All dep updates in one PR

Closes #1141 
Closes #1140

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,19 +139,19 @@
...
+    "eslint-plugin-jest": "23.0.0",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-prefer-object-spread": "1.2.1",
     "eslint-plugin-prettier": "3.1.1",
     "eslint-plugin-react": "7.16.0",
-    "eslint-plugin-react-hooks": "2.1.2",
-    "formik": "1.5.8",
-    "glob": "7.1.4",
+    "eslint-plugin-react-hooks": "2.2.0",
+    "formik": "2.0.2",
```
Comment (tdeekens) [inline package.json]: I assume you quickly checked the `resetForm` calls?

Comment (montezume) [reply package.json]: 😬

Comment (montezume) [reply package.json]: 😄

**Comment by montezume** [comment]
I'm going to investigate the Percy diffs and make sure all the Formik resets are working.

**Pull request merged**